### PR TITLE
Create general-use panel macro with a toggling visual caret

### PR DESCRIPTION
**Pull request by samwilson**
Generalize the panel that currently is only being used for event
participants, to remove event-specific stuff and also add a
visual caret to the header to indicate more clearly that the
panel can be collapsed.

Bug: https://phabricator.wikimedia.org/T200551

### CODE REVIEW

File: app/Resources/assets/css/application.scss [hunk 1]
```diff
@@ -219,3 +219,25 @@ a.disabled {
...
+.panel {
+    .panel-heading {
+        display: flex;
+        align-items: end;
+        .panel-title {
+            font-size: 24px;
+            flex: 3;
+        }
+        .panel-description {
+            flex: 1;
```
Comment (MusikAnimal) [inline app/Resources/assets/css/application.scss]: Only thing here is the text isn't vertically aligned. I accomplished this by setting an explicit `height` and `line-height` https://github.com/wikimedia/grantmetrics/pull/87/files#diff-ff1c57142581996a6cd7e089b450dc9fL112 so we might restore that.

There may be a better way to do it, though!

Comment (samwilson) [reply app/Resources/assets/css/application.scss]: I was going for the `end` alignment. But now you mention it, `center` is better! :) And no need for explicit height.

Oh, and it turns out that I'm confused: the [W3C]( https://www.w3.org/TR/css-flexbox-1/#propdef-align-items) says different things to [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/align-items). Anyway, center is what we want yeah?

**Comment by samwilson** [comment]
Don't merge yet; I'm adding the tooltip to the panel title.

File: app/Resources/assets/css/application.scss (hunk 1 above)
Comment (MusikAnimal) [reply app/Resources/assets/css/application.scss]: Yeah vertically centered, right-alighted.

Comment (samwilson) [reply app/Resources/assets/css/application.scss]: Done. (Sorry, I did the gerrit-world thing of modifying an existing commit, so GH probably didn't tell you about it.)

**Comment by samwilson** [comment]
Ready for review now.

**Comment by samwilson** [comment]
I forgot the qqq messages. Done now.

**Comment by MusikAnimal** [comment]
Looks fantastic!

**Pull request merged**